documentation/01_tutorial/11-10-ui-and-combat.html.md

```diff
@@ -200,13 +200,13 @@ This all seems simple enough, but it's actually going to require several compone
 
 	At the very top, before doing anything else in that function.
 
-    You will also need to add
+	You will also need to add the following line to the top of both the `Enemy` and `PlayState` classes.
```

Comment:
This should be mentioned where changes are made to the `PlayState` class, it doesn't make sense in this step.